src/Common/APICollection.php

```diff
@@ -88,6 +88,16 @@ public function links($presenter)
     }
 
     /**
+     * Get the first item from the collection.
+     *
+     * @return APIResponse|null
+     */
+    public function first()
+    {
+        return $this->count() > 0 ? $this->items[0] : null;
```

Comment:
Is the `> 0` needed? If it's 0 won't it be false-y otherwise a truthy integer? Not a big deal and fine if best to have for clarity or making sure it's a boolean. Just curious :)